python/pyspark/sql/functions.py

```diff
@@ -1195,12 +1195,12 @@ def mode(col: "ColumnOrName") -> Column:
     ...     ("dotNET", 2013, 48000), ("Java", 2013, 30000)],
     ...     schema=("course", "year", "earnings"))
     >>> df.groupby("course").agg(mode("year")).show()
-    +------+----------+
-    |course|mode(year)|
-    +------+----------+
-    |  Java|      2012|
-    |dotNET|      2012|
-    +------+----------+
+    +------+-----------------+
+    |course|mode(year, false)|
```

Comment:
Shell we support new parameter in Python API too?

Reply:
Thanks! This is actually a good question and I think we shall.
I've added the new param to both Scala and Python APIs in both SQL and Connect in the latest.